web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -451,11 +448,48 @@ describe("<LicenseTask />", () => {
       it("displays NOT_FOUND error alert when license status cannot be found", async () => {
```

Comment:
We should have a test at the take page level where you have license data for license A, but visiting the task page for license B, the check had been done so you see the hot found error